packages/swapper/src/swappers/ArbitrumBridgeSwapper/utils/fetchArbitrumBridgeSwap.ts

```diff
@@ -40,13 +41,19 @@ export const fetchArbitrumBridgeSwap = async ({
   receiveAddress,
   supportsEIP1559,
   assertGetEvmChainAdapter,
-}: FetchArbitrumBridgeSwapInput): Promise<{
+  priceOrQuote,
+}: FetchArbitrumBridgeSwapInput<T>): Promise<{
   request:
     | Omit<ParentToChildTransactionRequest | ChildToParentTransactionRequest, 'retryableData'>
     | undefined
   allowanceContract: string
   networkFeeCryptoBaseUnit: string
 }> => {
+  if (priceOrQuote === 'quote' && !receiveAddress)
+    throw new Error('receiveAddress is required for Arbitrum Bridge quotes')
+  if (priceOrQuote === 'quote' && !sendAddress)
+    throw new Error('sendAddress is required for Arbitrum Bridge quotes')
```

Comment:
This shouldn't be an issue at types-level given the generic `priceOrQuote` T added, but safety never hurts